src/kOS/Function/Misc.cs

```diff
@@ -28,6 +28,51 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+[Function("hudtxt")]
+    public class FunctionHudTxt : FunctionBase
+    {
+
+        public override void Execute(SharedObjects shared)
+
+        {
+            int mirror          = Convert.ToInt32(shared.Cpu.PopValue());
+            string color        = shared.Cpu.PopValue().ToString();
```

Comment:
I'd prefer this to use the already-existing kOS RGBA color type, for proper coherency in the mod.

i.e.  HUDTXT( "pink text",  6, 1, 2, RGB(1,0.7,0.7), false).
or.  HUDTXT( "pink text",  6, 1, 2, red, false).  // no quotes around 'red' because it's already a built-in term in kOS.

(In which case, you'd be expecting to pop an object of type kOS.Suffixed.RgbaColor from the stack here.)